eslint.config.js

```diff
@@ -99,6 +99,12 @@ export default tseslint.config(
 			"object-shorthand": "error",
 		},
 	}),
+	{
+		files: ["package.json"],
+		rules: {
+			"package-json/sort-collections": "off",
+		},
+	},
```

Comment:
Irksome conflict between the ESLint plugin and Prettier. I don't have the bandwidth to investigate right now.